catalog-info.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: backstage.io/v1alpha1
+kind: System
+metadata:
+  name: mozilla-monitor
+  annotations:
+    github.com/project-slug: mozilla/blurts-server
+    github.com/team-slug: smozilla/teams/firefox-monitor-engineering
```

Comment:
@mansaj Heh, I was curious because I never heard of Backstage so I took a look at the PR - I assume the `s` here is not intentional?